collect_app/src/test/java/org/odk/collect/android/widgets/BarcodeWidgetTest.java

```diff
@@ -183,6 +185,23 @@ public void clickingBarcodeButton_whenFrontCameraIsAvailable_launchesCorrectInte
         assertThat(shadowActivity.getNextStartedActivity().getBooleanExtra(Appearances.FRONT, false), is(true));
     }
 
+    @Test
+    public void whenPromptHasNoAnswerAppearance_answerIsNotDisplayed() {
+        FormEntryPrompt prompt = new MockFormEntryPromptBuilder(promptWithAppearance(Appearances.HIDDEN_ANSWER))
+                .withAnswer(new StringData("original contents"))
+                .build();
+
+        BarcodeWidget widget = createWidget(prompt);
+        assertThat(widget.binding.barcodeAnswerText.getVisibility(), equalTo(View.GONE));
```

Comment:
Why do you mix `is` and `equalTo` methods instead of using just one and the same?

Reply:
Ah that's me copy pasting assertions from other tests. Should we update our Hamcrest guidelines (in `CODE-GUIDLINES.md`) to encourage `equalTo` rather than `is`? Now we're writing all new code in Kotlin, `is` doesn't really work as a default anymore (due to it needing escaped).

Reply:
I think it's a good idea I haven't used `is` at all for a while.